_security/access-control/api.md

```diff
@@ -235,9 +261,10 @@ Introduced 1.0
 
 #### Request
 
-```
+```json
```

Comment:
qq, does changing the format to json help with the layout? I was fiddling around with this when I created the issue with `json`, `shell`, `code`, etc. but it just ended up being displayed as a normal code block.

Reply:
@DarshitChanpura It doesn't help with the layout, but it does provide syntax highlighting.

Reply:
Yeah, good question. The team just started using a linter (Vale) to help analyze style issues in the file. Eventually the linter/code analyzer will flag a code block if it doesn't have a language associated with it. So I added the `json` language to the examples to get ahead of what's coming down the road.